docs/develop/platform-specific-requirement-sets.md

```diff
@@ -0,0 +1,126 @@
+
+### Code
+
+Another option is to implement a runtime check in your code. This way, you can make new features available to your customers on those platforms. The runtime check also ensures that the platform-specific code doesn't run on unsupported platforms and cross-platform features continue to work for your customers. The following code is an example that checks for a specific requirement set.
+
+```javascript
+if (Office.context.requirements.isSetSupported("WordApiDesktop", "1.1")) {
+   // Any API exclusive to this WordApiDesktop requirement set.
+}
+```
+
+Whenever platform-specific APIs become available cross-platform, enable your customers on all supported platforms to use those features by implementing one of the following options.
+
+- Remove the runtime check. But note that customers on older Office clients, especially on Windows, may hit errors if their client doesn't support the new APIs yet.
+- Update the runtime code to check for the cross-platform requirement set.
+
+A variation is to do a runtime check for a particular API. This means that the encapsulated code should run on any platforms that support that API. If the API was first released in a platform-specific requirement set then promoted to a cross-platform one, you shouldn't need to update your code unless you made assumptions about the supported platforms. The following code is an example.
+
+```javascript
+if (Office.context.document.setSelectedDataAsync)
+{
+    // Run code that uses document.setSelectedDataAsync.
+}
+```
+
+## Notify customers on AppSource
+
+If your add-in is in AppSource or the Office store, be sure to notify customers about any platform-specific behavior.
+
+**Details + support** > **Products supported** on your add-in's AppSource page should automatically show the appropriate supported platforms based on what requirements you declared in the manifest.
+
+However, if your add-in is supported cross-platform but you also implemented platform-specific behaviors, you should point out those feature differences in the **Overview** section on your add-in's AppSource page.
+
+## Exceptions
+
+The following are exceptions to the approach described.
+
+### Online-only requirement sets
+
+An online-only requirement set is a superset of the latest numbered requirement set. For each Office application with an online-only requirement set, `1.1` is the only version. It's invalid to specify an online-only requirement set in the [Set element](/javascript/api/manifest/set) of your add-in manifest.
+
+To check for APIs that are only supported in these requirement sets and to prevent your add-in from trying to run the code on unsupported platforms, you should add code similar to the following:
```

Comment:
```suggestion
To check for APIs that are only supported in these requirement sets and to prevent your add-in from trying to run the code on unsupported platforms, you should add code similar to the following.
```

Reply:
@lindalu-MSFT I have a colon here because of [this style guide entry](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/f/following)